homeassistant/helpers/floor_registry.py

```diff
@@ -229,6 +233,13 @@ async def async_load(self) -> None:
 
         if data is not None:
             for floor in data["floors"]:
+                if TYPE_CHECKING:
+                    assert isinstance(floor["aliases"], list)
+                    assert isinstance(floor["icon"], str)
+                    assert isinstance(floor["level"], int)
+                    assert isinstance(floor["name"], str)
+                    assert isinstance(floor["floor_id"], str)
+
```

Comment:
This is to make mypy happy, I am open to a better approach. Under normal circumstances, there should not be any invalid data.

Reply:
I think it's ok for now. Hopefully we can improve it later, with later versions of mypy, with specific types per key in the dict.